packages/custom_lint/lib/custom_lint.dart

```diff
@@ -82,46 +83,42 @@ Future<void> _runPlugins(
     exitCode = 1;
     stderr.writeln('$err\n$stack');
   }
-
-  // Since no problem happened, we print a message saying everything went well
-  if (exitCode == 0) {
-    stdout.writeln('No issues found!');
-  }
 }
 
 void _renderLints(
   List<AnalysisErrorsParams> lints, {
   required Directory workingDirectory,
 }) {
-  lints.sort(
-    (a, b) => a
-        .relativeFilePath(workingDirectory)
-        .compareTo(b.relativeFilePath(workingDirectory)),
-  );
-
-  for (final lintsForFile in lints) {
-    final relativeFilePath = lintsForFile.relativeFilePath(workingDirectory);
-
-    lintsForFile.errors.sort((a, b) {
-      final lineCompare = a.location.startLine.compareTo(b.location.startLine);
-      if (lineCompare != 0) return lineCompare;
-      final columnCompare =
-          a.location.startColumn.compareTo(b.location.startColumn);
-      if (columnCompare != 0) return columnCompare;
-
-      final codeCompare = a.code.compareTo(b.code);
-      if (codeCompare != 0) return codeCompare;
-
-      return a.message.compareTo(b.message);
-    });
+  final errors = lints
+      .expand(
+        (lintsForFile) => lintsForFile.errors
```

Comment:
I don't like the syntax. Too much going on at once.

Why sort the errors twice? Why not have the `..sort` logic in `sortedBy`?

Reply:
Good point, I refactored it and now also don't need the `collection` package.

Reply:
I can extract the `expand(...)` into a new variable if you prefer this.